config/locales/idv/en.yml

```diff
@@ -10,6 +10,10 @@ en:
       incorrect_password: The password you entered is not correct.
       invalid_ccn: Credit card number should be only last 8 digits.
       missing_finance: You must provide a financial account number.
+      pattern_mismatch:
```

Comment:
I think you are going to have to add placeholders for these to the `es.yml` in this dir :)